lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -919,6 +919,92 @@ setReadOnly( Complex64Array.prototype, 'every', function every( predicate, thisA
 	return true;
 });
 
+/**
+* Returns the modified typed array filled with fill value.
+*
+* @name fill
+* @memberof Complex64Array.prototype
+* @type {Function}
+* @param {Complex64} value - fill value
+* @param {integer} [start=0] - starting index (inclusive)
+* @param {integer} [end] - ending index (exclusive)
+* @throws {TypeError} `this` must be a complex number array
+* @throws {TypeError} `value` must be a complex number
+* @throws {TypeError} `start` must be an integer
+* @throws {TypeError} `end` must be an integer
```

Comment:
```suggestion
* @throws {TypeError} first argument must be a complex number
* @throws {TypeError} second argument must be an integer
* @throws {TypeError} third argument must be an integer
```

In general, we prefer "first argument" rather than the parameter name for maintainability reasons. If we rename the parameter, we don't want to accidentally forget to update the JSDoc error description.